packages/block-library/src/navigation/edit.js

```diff
@@ -171,6 +215,31 @@ function Navigation( {
 							{ __( 'Create from all top-level pages' ) }
 						</Button>
 
+						{ !! hasMenus && (
```

Comment:
We'll need to check for `selectedMenu` before we display this as the value prop on `SelectControl` element cannot be `null`.

Reply:
Fixed